toolbox/repo/validate_role_files.py

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env python
+
+import glob, os
+import yaml
+import pathlib
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
```

Comment:
nit: Maybe call this `SCRIPT_DIR` because `THIS_DIR` might potentially be confused with "Current directory"

Reply:
I use this name in the rest of the toolbox, so I'll keep the same name here for consistency